setup.cfg

```diff
@@ -55,7 +55,85 @@ test =
     hypothesis>=5.5.3
     pytest>=6.0
     pytest-xdist>=1.31
-
+# optional extras for recommended dependencies
+# see: doc/source/getting_started/install.rst
+recommended =
+    bottleneck>=1.3.2
+    numexpr>=2.7.1
+computation =
+    numba>=0.53.0
+    scipy >=1.7.1
+    xarray>=0.19.0
+s3 =
```

Comment:
IMO `aws` would be more general here

Reply:
None today, just aligning to how there's a `gcp` section.

Additionally this future proofs just in case there's an aws service in the future that pandas can talk to.